### ISSUE TITLE
Create a logo

### ISSUE THREAD

**Issue by s4wny** [sjchat/sjchat#17]
Create a logo that we can use for the project, website and the app.

**Comment by fredriksebek** [comment]
Started drafting icons: https://www.dropbox.com/sh/jaikw3a5a377gcm/AAB7Ay5FE8I9TYpFHqcL89kba?dl=0
It would be nice if people gave their opinions, so we can come up with something we all like!

**Comment by Krarl** [comment]
There seems to be a lot of pxm files, which seems very difficult to open unless you have a mac. Is there any more icons than the one you exported as a png? :)

**Comment by fredriksebek** [comment]
Absolutely! I will export the ones I find the most relevant and continue drafting!

**Comment by Krarl** [comment]
![SJChat](http://i.imgur.com/71Axksp.png)

My own more minimalistic idea. It's just the Oswald font from google and two 2px borders around it, so it could probably be created using only css.

**Comment by fredriksebek** [comment]
I do not mind, I can create the css file!

**Comment by fredriksebek** [comment]
Actually what is the point of using css? Maybe svg is better?

**Comment by s4wny** [comment]
@Krarl @fredriksebek I like the logo. But it would be nice to have a squared-sized logo too that we can use here at github as profile picture for the [organization](https://github.com/sjchat).
